webapp/app.py

```diff
@@ -166,6 +166,17 @@
 # Set up application
 # ===
 
+# Vanilla Macro loader example
+
+from jinja2 import ChoiceLoader, FileSystemLoader
+
+# ChoiceLoader attempts loading templates from each given path in successive
+# order
+loader = ChoiceLoader([
+    FileSystemLoader('templates'),
+    FileSystemLoader('node_modules/vanilla-framework/templates/')
+])
```

Comment:
What happens if a project has a file in `templates` with the exact same file path as a macro we publish that they'd like to use?

My assumption is the project's template will be used since it's listed first here. However this means we are slightly restricting the file structure of projects that use flask base. 

This isn't a bad thing, but maybe we should add some logic that outputs warnings if any identical file-paths are found between these two directories?

Reply:
> Yep, I'd been wondering the same thing - maybe the filenames of our macros should be namespaced somehow (`vf.tiered_list.jinja` or whatever)?

I'd support that.